dbt/tests/data_tests/generic_tests/expect_unique_combination.sql

```diff
@@ -0,0 +1,14 @@
+{% test expect_unique_combination(model, columns) %}
```

Comment:
This is an annoying suggestion, but maybe we should call this `expect_unique_column_combination`

Reply:
Will do